recipe/meta.yaml

```diff
@@ -21,11 +21,13 @@ requirements:
     - setuptools
 
   run:
-    - python >={{ python_min }}
-    - google-api-core >=1.31.6,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0
-    - google-auth >=1.25.0,<3.0dev
-    # grpc extra
-    - grpcio >=1.38.0,<2.0.0dev
+    # excluding python 3.14 for now because we don't have the required grpcio on conda-forge
+    - python >={{ python_min }},<3.14
+    - google-api-core >=1.31.6,<3.0.0,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0
+    - google-auth >=1.25.0,<3.0.0
+    # grpc extra, python < 3.14
+    - grpcio >=1.38.0,<2.0.0
+    - grpcio-status >=1.38.0,<2.0.0
```

Comment:
I'm not seeing movement on grpcio 1.75.1 or 1.76.0 on conda-forge (which is understandable -- it's a beast!).  I think the way forward for `google-cloud-*` for now has to be excluding python 3.14.